media-proxy/src/proxy_context.cc

```diff
@@ -765,11 +762,9 @@ int ProxyContext::TxStart(const TxControlRequest* request)
 
     if (mDevHandle == NULL) {
         struct mtl_init_params st_param = {};
-
-        ParseStInitParam(request, &st_param);
-
         /* set default parameters */
         st_param.flags = MTL_FLAG_BIND_NUMA;
```

Comment:
```suggestion
```
Same here: this assignment doesn't make sense because `st_param.flags` is being overwritten next line in `ParseStInitParam()`